app/src/main/java/com/amaze/filemanager/ui/dialogs/GeneralDialogCreation.java

```diff
@@ -163,6 +163,16 @@ public static MaterialDialog showNameDialog(
     WarnableTextInputLayout tilTextfield =
         dialogView.findViewById(R.id.singleedittext_warnabletextinputlayout);
 
+    tilTextfield.requestFocus();
+
+    textfield.postDelayed(
+        () -> {
+          InputMethodManager inputMethodManager =
+              (InputMethodManager) m.getSystemService(Context.INPUT_METHOD_SERVICE);
+          inputMethodManager.showSoftInput(textfield, InputMethodManager.SHOW_IMPLICIT);
+        },
+        100);
```

Comment:
Is a little hack, but it's okay.

Reply:
Whats up with this?

Reply:
@VishnuSanal why does this wait for 100 millis?

Reply:
> @VishnuSanal why does this wait for 100 millis?

To finish drawing... (Now I feel like I should've checked for nullity too!)

Reply:
Shouldn't you just  do it after the dialog.draw() then?

Reply:
> Shouldn't you just do it after the dialog.draw() then?

Didn't get you @EmmanuelMess

Reply:
If you wait 100 millis for the textfield to finish drawing, why don't you run the code after the dialog.show(), AFAIK, the dialog is assured to be showing after dialog.show().

Reply:
> If you wait 100 millis for the textfield to finish drawing, why don't you run the code after the dialog.show()? AFAIK, the dialog is assured to be showing after dialog.show().

For some unknown (& weird!) reason, it does not work when placed after `dialog.show()`!

Reply:
Wait, should we check for nullity of the view (inside postdelayed) - in case it's not drawn after 100 millis?